src/main/java/com/google/javascript/cl2dts/DeclarationGenerator.java

```diff
@@ -463,10 +484,12 @@ public Void caseStringType() {
 
         @Override
         public Void caseObjectType(ObjectType type) {
-          if (type.isRecordType()) {
+          if (type.isNativeObjectType()) {
```

Comment:
I guess, something that is not a Function, interface etc. I wonder if that can simplify our isDefault check (obviously, for the next time we revisit that code).